pagopa/pn-frontend
# PN-10843-fe-api-lista-notifiche

## Short Description
Migrate the old notification list API to the BFF's API.

## List of Changes Proposed in This Pull Request
- Generated models from the BFF OpenAPI.
- Updated `getSentNotifications()` and `getReceivedNotifications()` with the BFF's API.
- Updated related tests.
- Updated related mocks.

## How to Test
Start each portal (PA, PG & PF) and verify if you can see the list of notifications received/sent/delegated on the dashboard.
